pkg/kapp/logs/pod_container_log.go

```diff
@@ -44,6 +44,30 @@ func NewPodContainerLog(
 }
 
 func (l PodContainerLog) Tail(ui ui.UI, cancelCh chan struct{}) error {
+	for {
+		linePrefix := ""
+		if len(l.opts.LinePrefix) > 0 {
+			linePrefix = l.opts.LinePrefix + " | "
+		}
+		err := l.StartTail(ui, cancelCh)
+
+		if err == io.EOF {
+			if l.opts.Follow {
+				ui.BeginLinef("%s# container restarting '%s' logs\n", linePrefix, l.tag)
+				time.Sleep(500 * time.Millisecond)
+				continue
+			} else {
```

Comment:
We don't need and `else` after `continue`.

Reply:
`else` condition is for `l.opts.Follow`. continue is because I don't want to return from the function. Scenario is: if somebody has given `-f` option, then we should continuously try to get the stream

Reply:
I apologise for not being clear earlier, but what I meant was that, this
```go
for {
    if l.opts.Follow {
        //do something
        continue
    } else {
        //do something else
        return
    }
}
```
would behave same as this
 ```go
for {
    if l.opts.Follow {
        //do something
        continue
    }
    //do something else
    return
}
```
as `continue` at the end of `if` would have the same effect as that of an `else` statement.

Reply:
I think it is more clear to me as it is right now